src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1411,6 +1411,7 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
             var optimizations = compilation.Options.OptimizationLevel;
 
             ILBuilder builder = new ILBuilder(moduleBuilder, localSlotManager, optimizations, method.AreLocalsZeroed);
+            bool hasStackalloc = false;
```

Comment:
> = false [](start = 30, length = 8)

Consider removing initialization so that we can rely on the compiler to ensure that all code paths assign to the local.